.github/workflows/test_on_push.yaml

```diff
@@ -22,8 +22,10 @@ jobs:
       - uses: actions/checkout@master
 
       - name: Install requirements for community
+        uses: tarantool/setup-tarantool@v1
+        with:
+          tarantool-version: ${{ matrix.tarantool-version }}
         run: |
-          curl -L https://tarantool.io/installer.sh | sudo VER=${{ matrix.tarantool-version }} bash
           sudo apt install -y tarantool-dev
```

Comment:
tarantool-dev is installed by setup-tarantool.

Reply:
Removed `tarantool-dev` installation.